uikit/zoomcontrols/src/main/res/drawable/zoom_controls_button_background.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
ked ma ten layer-list len 1 polozku nie je to to iste ako iba ten shape osamote? ;)

Reply:
jj, pravdu máš, opravím, díky ;)